bin/app.ts

```diff
@@ -281,6 +281,9 @@ const jsonRpcProviders = {
   WEB3_RPC_56: process.env.JSON_RPC_PROVIDER_56!,
   WEB3_RPC_43114: process.env.JSON_RPC_PROVIDER_43114!,
   WEB3_RPC_8453: process.env.JSON_RPC_PROVIDER_8453!,
+  UNI_RPC_PROVIDER_INFURA_KEY: process.env.UNI_RPC_PROVIDER_INFURA_KEY!,
```

Comment:
I think we should name it more generically, because we might dynamically switch between providers at runtime.